tests/integration/test_multiple_response.py

```diff
@@ -425,52 +426,58 @@ def test_cat_x_mr_x_mr_proportions_by_cell(self):
+    def test_mr_x_cat_x_cat_by_col(self):
+        # TODO: Check expectations with Mike and Jon
         cube = CrunchCube(SELECTED_3_WAY_2)
         # Only compare 0 slice (parity with whaam tests)
         expected = np.array([
             [0.5923110874002918, 0.3758961399306439],
             [0, 0],
             [0.49431928922535223, 0.6091963925363675]
         ])
-        actual = cube.proportions(axis=2)[0]
-        # Since cube is 3D, row dim is 2 (instead of 1)
+        # 3D cube => col == 1
+        actual = cube.proportions(axis=1)[0]
         np.testing.assert_almost_equal(actual, expected)
 
     def test_cat_x_mr_x_cat_by_col(self):
-        cube = CrunchCube(SELECTED_3_WAY)
-        # Only take first slice (parity with whaam tests).
-        expected = np.array([[1, 0], [1, 0], [1, 0]])
-        # Since MR is 2nd dim, the cube is considered 2 dimensional,
-        # and the axis 0 represents column direction.
-        actual = cube.proportions(axis=0)[0]
-        np.testing.assert_almost_equal(actual, expected)
-
-    def test_cat_x_mr_x_cat_by_row(self):
+        # TODO: Check expectations with Mike and Jon
         cube = CrunchCube(SELECTED_3_WAY)
         # Only take first slice (parity with whaam tests).
         expected = np.array([
             [0.0997975162008577, np.nan],
             [0.20327963774693497, np.nan],
             [0.3113417143573762, np.nan],
         ])
-        # Since MR is 2nd dim, the cube is considered 2 dimensional,
-        # and the axis 1 represents row direction.
+        # 3D cube => col == 1
         actual = cube.proportions(axis=1)[0]
         np.testing.assert_almost_equal(actual, expected)
 
+    def test_cat_x_mr_x_cat_by_row(self):
+        # TODO: Check expectations with Mike and Jon
+        cube = CrunchCube(SELECTED_3_WAY)
+        # Only take first slice (parity with whaam tests).
+        expected = np.array([[1, 0], [1, 0], [1, 0]])
+        # 3D cube => row == 2
+        actual = cube.proportions(axis=2)[0]
+        np.testing.assert_almost_equal(actual, expected)
+
     def test_cat_x_mr_x_cat_by_cell(self):
+        # TODO: Check expectations with Mike and Jon
         cube = CrunchCube(SELECTED_3_WAY)
         # Only take first slice (parity with whaam tests).
         # TODO: Check with @jonkeane, since R results are slightly
         # different. (It's using (0, 2) rather than (1, 2) axis).
         expected = np.array([
-            [0.03326584, 0],
-            [0.06775988, 0],
-            [0.10378057, 0],
+            [0.0997975162008577, 0],
+            [0.20327963774693497, 0],
+            [0.3113417143573762, 0],
         ])
-        # Since MR is 2nd dim, the cube is considered 2 dimensional,
-        # and the axis None represents cell direction.
+        # Old expectation:
+        # expected = np.array([
+        #     [0.03326584, 0],
+        #     [0.06775988, 0],
+        #     [0.10378057, 0],
+        # ])
         actual = cube.proportions(axis=None)[0]
```

Comment:
Those look good, and similarly the second (first if you're 0 indexed) slice should be:

```
[0, 0.09558894744980956],
[0, 0.1880943257946565],
[0, 0.3991641311686333],
```